src/MSBuild.NetCoreDependencies/MSBuild.NetCoreDependencies.nuget.props

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(NuGetPackageRoot)' == ''">
+    <NuGetPackageRoot>C:\Users\daplaist\.nuget\packages\</NuGetPackageRoot>
```

Comment:
In other projects, we didn't check in this file.  I thought I added it to `.gitignore`, actually.  Do we need to check it in here?